source/agora/consensus/data/Transaction.d

```diff
@@ -97,6 +110,9 @@ public struct Transaction
 
     public void deserialize (scope DeserializeDg dg) nothrow @safe
     {
+        ubyte type_size;
+        deserializePart(type_size, dg);
```

Comment:
?

you're not storing it to this.type.

Reply:
Only digits are required for deserialize.